### PR TITLE
Use workflow if instead of bash if

### PR DESCRIPTION
**Pull request by roryabraham**
https://github.com/Expensify/Expensify.cash/pull/1809 pt. 2 with a safeguard that should hopefully not break the test.

### CODE REVIEW

File: .github/workflows/preDeploy.yml [hunk 1]
```diff
@@ -70,7 +70,8 @@ jobs:
 
         # TODO: Remove once we know that automerge doesn't infinitely loop with this action
       - name: Quit early
-        run: if [[ ${{ steps.checkCurrentVersion.outputs.CURRENT_VERSION == '1.0.2-60' }} ]]; then exit 1; fi;
+        if: ${{ steps.checkCurrentVersion.outputs.CURRENT_VERSION == '1.0.2-60 }}
```
Comment (Jag96) [inline .github/workflows/preDeploy.yml:73]: Missing a ' after the version here

Comment (roryabraham) [reply .github/workflows/preDeploy.yml]: fixed, good catch. Going too quick...

**Review by Jag96 — APPROVED** [review]

**Comment by Jag96** [comment]
Merging again to test workflows no need to wait for e2e tests here

**Comment by botify** [comment]
@Jag96 looks like this was merged without passing tests. Please add a note explaining why this was done or remove the `Emergency` label if this is not an emergency.

**Pull request merged**